api/chain.go

```diff
@@ -467,3 +476,34 @@ func (a *API) chainBlockByHashHandler(msg *apirest.APIdata, ctx *httprouter.HTTP
 	}
 	return ctx.Send(convertKeysToCamel(data), apirest.HTTPstatusOK)
 }
+
+// POST /chain/organizations/filter/page/<page>
+// returns a list of organizations paginated by the given page
+func (a *API) chainOrganizationsFilterPaginated(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
+	// get organizationId from the request body
+	var organizationId string
+	if err := json.Unmarshal(msg.Data, &organizationId); err != nil {
+		return fmt.Errorf("%w: %v", ErrCantParseOrgID, err)
```

Comment:
fmt.Errorf doesn't really work, this was my fault. i fixed it yesterday and just merged PR #832
 
example
https://github.com/vocdoni/vocdoni-node/pull/832/commits/a5f01b52c6493f10cdd76207cdcf9b2b323663e0#diff-8a1c97a3c76047a21682fceabcf8e05e4fdb9b8909d5e2209ec1fa626e153466L125-L126

new syntax is much prettier, tho :sunglasses: 

`return ErrCantParseOrgID.WithErr(err)`

Reply:
please replace all of the "return fmt.Errorf" inside all *Handler() func